hasher-matcher-actioner/hmalib/common/threatexchange_config.py

```diff
@@ -8,12 +8,14 @@
 from threatexchange.api import ThreatExchangeAPI
 from hmalib.common.logging import get_logger
 from hmalib.common import config as hmaconfig
-from hmalib.lambdas.fetcher import ThreatExchangeConfig
+from hmalib.common.config import HMAConfig
+from hmalib.common.fetcher_models import ThreatExchangeConfig
 from botocore.exceptions import ClientError
 
 logger = get_logger(__name__)
 FETCHER_ACTIVE = True
 WRITE_BACK = True
+MATCHER_ACTIVE = True
```

Comment:
Can we make these defaults? eg. MATCHER_ACTIVE_DEFAULT

Reply:
sure will do